src/runtime/components/Pagination.vue

```diff
@@ -167,9 +167,9 @@ const ui = computed(() => tv({ extend: tv(theme), ...(appConfig.ui?.pagination |
           </slot>
         </PaginationListItem>
 
-        <PaginationEllipsis v-else :key="item.type" :index="index" as-child :class="ui.ellipsis({ class: props.ui?.ellipsis })">
+        <PaginationEllipsis v-else :key="item.type" as-child :class="ui.ellipsis({ class: props.ui?.ellipsis })">
```

Comment:
Another thing, is `:key="item.type"` shouldn't be `:key="index"`? 🤔

```suggestion
        <PaginationEllipsis v-else :key="index" as-child :class="ui.ellipsis({ class: props.ui?.ellipsis })">
```